cdc/processor/sinkmanager/manager_impl.go

```diff
@@ -292,14 +295,17 @@ func (m *ManagerImpl) RemoveTable(tableID model.TableID) error {
 			zap.String("changefeed", m.changefeedID.ID),
 			zap.Int64("tableID", tableID))
 	}
+	// Awake all workers to check if the table sink is removed.
+	m.memQuota.awakeAll()
```

Comment:
Seems should close the table sink before awake all.

Reply:
Yes. Good catch!